net.pdl2ork.PdL2Ork.yml

```diff
@@ -0,0 +1,197 @@
+id: net.pdl2ork.PdL2Ork
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+default-branch: stable
+command: pd-wrapper.sh
+finish-args:
```

Comment:
It crashes immediately on opening Help> Show Devtools

It seems that it needs `--allow=devel` for ptrace but that doesn't seem to fix the crash.

Reply:
Those would be nw.js dev tools. Could be a combo of 23.08 and nw.js 0.67.1 which is the only one that consistently works across Linux, Mac, and Windows with minimal idiosyncrasies.

Reply:
I will try to rebuild wtih nwjs-sdk instead of the nwjs library to see if this fixes it.

Reply:
So, it appears that using nwjs-sdk instead of nwjs has fixed things. I just pushed the update that should hopefully address all the questions you raised.